plugins.json

```diff
@@ -1869,6 +1869,23 @@
         "title": "Telegram",
         "version": "1.3.3"
     },
+    "ThemeMaestro": {
+        "author": "JustFxDev (Fx)",
+        "compatible_version": ">=1.2.11",
+        "description": "Nice and colored theme family for Kanboard. Supports the Customizer plugin.",
+        "download": "https://github.com/JustFxDev/ThemeMaestro/blob/main/release/v0.7.0/ThemeMaestro.zip",
```

Comment:
I'm not sure if this link will install or parse correctly as opening it should trigger a download, but instead it goes to the GitHub page.

I think the file structure is wrong for this plugin. Check the structure in the readme of this repository.